mlir/lib/Dialect/MIOpen/Transforms/LowerMIOpenOps.cpp

```diff
@@ -15,51 +15,32 @@
 // limitations under the License.
 // =============================================================================
 //
-// This pass converts miopen.conv2d into miopen.transform and
-// miopen.gridwise_gemm.
+// These passes convert the MIOpen threadwise ops into constructs from the
+// rest of MLIR so that they can be lowered to the GPU and LLVM dialects.
```

Comment:
Maybe `ThreadwiseGemmToStd.cpp` to be consistent with the rest of the file names? `LowerMIOpenOps.cpp` seems to generic compared to what it does now.

Reply:
... good point, I can do that.